Makefile

```diff
@@ -242,8 +242,13 @@ local-build: ## Build all container images necessary to run the whole operator
 	$(CONTAINER_TOOL) build -v $(GO_CONTAINER_CACHE):/go:z -f Dockerfile.mrvlVSP.rhel -t $(MARVELL_VSP_IMAGE)
 	$(CONTAINER_TOOL) build -v $(GO_CONTAINER_CACHE):/go:z -f Dockerfile.IntelVSP.rhel -t $(INTEL_VSP_IMAGE)
 
+.PHONE: prepare-multi-arch
+prepare-multi-arch:
+	test -f /proc/sys/fs/binfmt_misc/qemu-aarch64 || sudo podman run --rm --privileged quay.io/bnemeth/multiarch-qemu-user-static --reset -p yes
```

Comment:
Looking at this a bit more, you are doing "prepare" step but it is not obvious where it is being used. Is there a specific ARM container you are trying to run on x86 build?

Reply:
the prepare-multi-arch is a pre-req for the `local-buildx` target. This command needs to be executed to prepare buildah for the multi-arch build.

In the past we were lucky in that we didn't have `RUN` commands in the second part of our containers. Also, this will be a requirement for 4.19 going forward since they changed the builder layer image to be multi-arch too (that used to be x86 only, and we cross-compiled from x86 to arm in that layer _by chance_).